src/test/java/com/xceptance/xlt/engine/CachingHttpWebConnectionTest.java

```diff
@@ -204,7 +193,7 @@ public void testExpires_NoCachingHints() throws Exception
         Assert.assertEquals(0, time);
     }
 
-    private String formatDate(final long time)
+    private static String formatDate(final long time)
     {
         return new SimpleDateFormat("EEE, d MMM yyyy HH:mm:ss z", Locale.ENGLISH).format(new Date(time));
```

Comment:
Reuse date formatter of CachingHttpWebConnection once it declares such a constant?

Reply:
Done.